### ISSUE TITLE
Typo in inlineCallbacks documentation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7643]
|<img alt="trevorcreech's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| trevorcreech reported|
|-|-|
|Trac ID|trac#7643|
|Type|defect|
|Created|2014-09-16 17:41:22Z|
|Branch|https://github.com/twisted/twisted/tree/inlinecallbacks-docs-typo-7643|

https://twistedmatrix.com/documents/current/api/twisted.internet.defer.inlineCallbacks.html has incorrect capitalization (inlineCallBacks instead of inlineCallbacks) in the first example.

Attachments:

* [t7643.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a83/a83954a74fa4a3b76dc8aa4a43c5fcb445c3378f/d1875ca0123d331ddf07b908eb41177c7da7ddff.diff) (525 bytes) - added by Alex on 2014-09-16 17:42:35Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7643 7643
type__defect defect
reporter__trevorcreech trevorcreech
priority__normal normal
milestone__ 
branch__branches_inlinecallbacks_docs_typo_7643 branches/inlinecallbacks-docs-typo-7643
branch_author__khorn khorn
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1410889282190956 1410889282190956
changetime__1411100200227668 1411100200227668
version__None None
owner__khorn khorn

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex set owner to @glyph|
|-|-|

Attached patch fixes the issue.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @khorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_3"></a>|@khorn commented|
|-|-|

(In [43135]) Branching to inlinecallbacks-docs-typo-7643.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_4"></a>|@khorn commented|
|-|-|

(In [43136]) Fix typo in inlineCallbacks documentation. Refs [#7643](https://github.com/twisted/twisted/issues/7643)

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_5"></a>|@khorn commented|
|-|-|

Looks good.  Merging.

(Probably should have had a news fragment though, but I went ahead and added it.)

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_6"></a>|@khorn set status to `closed`|
|-|-|

(In [43137]) Merge inlinecallbacks-docs-typo-7643: Fix typo in inlincallbacks documentation.

Author: Alex
Reviewer: khorn
Fixes: [#7643](https://github.com/twisted/twisted/issues/7643)

This fixes a capitalization typo in the docsting for inlineCallbacks.
